web/app/scripts/views/duplicates/duplicates-list-controller.js

```diff
@@ -0,0 +1,101 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function DuplicatesListController($scope, $rootScope, $log, $modal, $state, AuthService,
+                                      InitialState, WebConfig, Duplicates, RecordState,
+                                      RecordSchemaState ) {
+        var ctl = this;
+        ctl.currentOffset = 0;
+        ctl.numDuplicatesPerPage = WebConfig.record.limit; //Just use the records limit
+        ctl.getPreviousDuplicates = getPreviousDuplicates;
+        ctl.getNextDuplicates = getNextDuplicates;
+        ctl.showResolveModal = showResolveModal;
+        ctl.userCanWrite = false;
+
+        InitialState.ready().then(init);
+
+        function init() {
+            ctl.isInitialized = false;
+            ctl.userCanWrite = AuthService.hasWriteAccess();
+
+            RecordState.getSelected().then(function(selected) { ctl.recordType = selected; })
+                .then(loadRecordSchema)
+                .then(loadDuplicates);
+        }
+
+        function loadRecordSchema() {
+            /* jshint camelcase: false */
+            var currentSchemaId = ctl.recordType.current_schema;
+            /* jshint camelcase: true */
+
+            return RecordSchemaState.get(currentSchemaId)
+                .then(function(recordSchema) {
+                    ctl.recordSchema = recordSchema;
+                });
+        }
+
+        /*
+         * Loads a page of duplicates from the API
+         * @param {int} offset Optional offset value, relative to current offset, used
+         *                     for pulling paginated results. May be positive or negative.
+         * @return {promise} Promise to load duplicates
+         */
+        function loadDuplicates(offset) {
+            var paramsOffset;
+            if (offset) {
+                ctl.currentOffset += offset;
+                if (ctl.currentOffset) {
+                    paramsOffset = ctl.currentOffset;
+                }
+            } else {
+                ctl.currentOffset = 0;
+                paramsOffset = 0;
+            }
+
+            /* jshint camelcase: false */
+            return Duplicates.query({record_type: ctl.recordType.uuid,
```

Comment:
To answer your question though - I can see a pretty good usecase for allowing filtering duplicates by record_type.  We might actually want to add it as a RecordDuplicate field to make things easier if that's a usecase we want to support.